lib/rules/no-unregistered-components.js

```diff
@@ -62,6 +62,9 @@ module.exports = {
         properties: {
           ignorePatterns: {
             type: 'array'
+          },
+          ignoreRecursive: {
```

Comment:
I didn't know that I could use self component in `<template>` using the same name as `name` option.
I don't need the this option because I think it is false positives and a bug.